src/Exporter/ReferentPersonLinkExport.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+namespace AppBundle\Exporter;
+
+use AppBundle\Entity\ReferentOrganizationalChart\ReferentPersonLink;
+use PhpOffice\PhpSpreadsheet\Spreadsheet;
+use PhpOffice\PhpSpreadsheet\Writer\Xlsx;
+use Symfony\Component\HttpFoundation\Response;
+
+class ReferentPersonLinkExport
+{
+    /**
+     * @param ReferentPersonLink[] $referentPersonLinks
+     *
+     * @return Spreadsheet
+     */
+    public function exportToXlsx(array $referentPersonLinks): Spreadsheet
+    {
+        $spreadsheet = new Spreadsheet();
+        $sheet = $spreadsheet->getActiveSheet();
+        $sheet->fromArray(['Référent', 'Zones', 'Prénom', 'Nom', 'Status', 'Email', 'Téléphone', 'Adresse']);
+        $sheet->fromArray($this->arrayObjectToArrayValues($referentPersonLinks), null, 'A2');
+
+        return $spreadsheet;
+    }
+
+    /**
+     * @param ReferentPersonLink[] $referentPersonLinks
+     *
+     * @return array[]
+     */
+    private function arrayObjectToArrayValues(array $referentPersonLinks): array
+    {
+        $result = [];
+
+        foreach ($referentPersonLinks as $referentPersonLink) {
+            $tab = [];
+
+            $tab[] = $referentPersonLink->getReferent()->getFullName();
+            $tab[] = $referentPersonLink->getReferent()->getAreasToString();
+            $tab[] = $referentPersonLink->getFirstName();
+            $tab[] = $referentPersonLink->getLastName();
+            $tab[] = $referentPersonLink->getPersonOrganizationalChartItem()->getLabel();
+            $tab[] = $referentPersonLink->getEmail();
+            $tab[] = $referentPersonLink->getPhone();
+            $tab[] = $referentPersonLink->getPostalAddress();
+
+            $result[] = $tab;
+        }
+
+        return $result;
+    }
+
+    public function createResponse(Spreadsheet $spreadsheet): Response
+    {
+        $writer = new Xlsx($spreadsheet);
+        ob_start();
```

Comment:
It looks so hacky, the lib does not provide any way to achieve the same exact thing ?

Reply:
I didn't find an other way :s.

Reply:
ok